topi/python/topi/nn/upsampling.py

```diff
@@ -20,7 +20,7 @@
 from ..util import simplify
 
 
-def upsampling(data, scale, layout="NCHW", method='NEAREST_NEIGHBOR'):
+def upsampling(data, scale, layout="NCHW", method='NEAREST_NEIGHBOR', align_corners=False):
```

Comment:
What about also changing methods here to lower cased

Reply:
You're right, that's better. Now updated.